AudioPlayer/AudioPlayer/player/extensions/AudioPlayer+PlayerEvent.swift

```diff
@@ -27,8 +27,8 @@ extension AudioPlayer {
             pausedForInterruption = true
             pause()
 
-        case .interruptionEnded where pausedForInterruption:
-            if resumeAfterInterruption {
+        case .interruptionEnded(let shouldResume) where pausedForInterruption:
+            if resumeAfterInterruption && shouldResume {
```

Comment:
resumeAfterInterruption is a configuration for clients, if they want to enable the resume feature
So in my opinion it is distinct enough to check them both together like this.